### PR TITLE
https://secure.php.net => https://www.php.net

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

https://secure.php.net => https://www.php.net
http://example.com/en/ => https://example.com/en/
http://www.w3.org/TR/xhtml1/#docconf => https://www.w3.org/TR/xhtml1/#docconf

### Testing Instructions

review

### Expected result

https://www.php.net

### Actual result

https://secure.php.net

### Documentation Changes Required

none

### CODE REVIEW

**Comment by brianteeman** [comment]
Why the change from secure.php.net  to www?

**Comment by HLeithner** [comment]
Secure.php.net is obsolete since php changed it server infrastructure and disabled all mirrors.

File: libraries/src/Filesystem/FilesystemHelper.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ class FilesystemHelper
 	 *
 	 * @return  mixed
 	 *
-	 * @link    https://secure.php.net/manual/en/function.filesize.php#71098
+	 * @link    https://www.php.net/manual/en/function.filesize.php#71098
```
Comment (ReLater) [inline libraries/src/Filesystem/FilesystemHelper.php]: I think `#71098` can be removed. There is no anchor 71098 on that page.

**Comment by brianteeman** [comment]
@HLeithner Thanks

**Comment by ReLater** [comment]
I have tested this item :white_check_mark: successfully on 45162e244388a95aa6f07fe803fc9726b5a719f1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24613">issues.joomla.org/tracker/joomla-cms/24613</a>.</sub>

File: libraries/src/Filesystem/FilesystemHelper.php (hunk 1 above)
Comment (Quy) [inline libraries/src/Filesystem/FilesystemHelper.php]:
```suggestion
	 * @link    https://www.php.net/manual/en/function.filesize.php
```

Comment (zero-24) [reply libraries/src/Filesystem/FilesystemHelper.php]: Should be fixed now.

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 7f55b0e45e9938b4e63e53e10d7e0acff27061ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24613">issues.joomla.org/tracker/joomla-cms/24613</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC, thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24613">issues.joomla.org/tracker/joomla-cms/24613</a>.</sub>

**Comment by HLeithner** [comment]
thx

**Pull request merged**